metatensor-torch/src/atomistic/model.cpp

```diff
@@ -1032,6 +1034,29 @@ static std::unordered_map<std::string, Quantity> KNOWN_QUANTITIES = {
         {"J", "Joule"},
         {"Ry", "Rydberg"},
     }}},
+    {"energy_ensemble", Quantity{/* name */ "energy", /* baseline */ "eV", {
+        {"eV", 1.0},
+        {"meV", 1000.0},
+        {"Hartree", 0.03674932247495664},
+        {"kcal/mol", 23.060548012069496},
+        {"kJ/mol", 96.48533288249877},
+        {"Joule", 1.60218e-19},
+        {"Rydberg", 0.07349864435130857},
+    }, {
+        // alternative names
+        {"J", "Joule"},
+        {"Ry", "Rydberg"},
+    }}},
+    {"dipole", Quantity{/* name */ "dipole", /* baseline */ "D", {
+        {"Debye", 1.0},
+        {"Coulomb-meter", 1000.0},
+        {"atomic units", 0.03674932247495664},
+    }, {
+        // alternative names
+        {"D", "Debye"},
+        {"C-m", "Coulomb-meter"},
```

Comment:
```suggestion
        {"Cm", "Coulombmeter"},
```

This seems to be the SI unit here